src/Core/Theme.re

```diff
@@ -1,366 +0,0 @@
-/*
- * Theme.re
- *
- * Theming color / info
- */
-
-// VSCode theme defaults and fallbacks defined in
-// https://github.com/microsoft/vscode/blob/634068a42471d610453d97fa0c81ec7e713c4e17/src/vs/platform/theme/common/colorRegistry.ts
-// https://github.com/microsoft/vscode/blob/abb01b183d450a590cbb78acb4e787eec5445830/src/vs/workbench/common/theme.ts
-// https://github.com/microsoft/vscode/blob/d49c5f3bc73ca4b41fe1306b2bf1d5b4bff96291/src/vs/editor/common/view/editorColorRegistry.ts
-// https://github.com/microsoft/vscode/blob/0991720b7b44ffc15760b578b284caff78ccf398/src/vs/workbench/contrib/terminal/common/terminalColorRegistry.ts
-
-open Revery;
-
-type defaults = {
-  foreground: Color.t,
-  editorBackground: Color.t,
-  editorForeground: Color.t,
-  editorIndentGuideBackground: Color.t,
-  editorIndentGuideActiveBackground: Color.t,
-  editorWhitespaceForeground: Color.t,
-};
-
-let light: defaults = {
-  foreground: Color.hex("#616161"),
-  editorBackground: Color.hex("#fffffe"),
-  editorForeground: Color.hex("#333"),
-  editorIndentGuideBackground: Color.hex("#D3D3D3"),
-  editorIndentGuideActiveBackground: Color.hex("#939393"),
-  editorWhitespaceForeground: Color.hex("#33333333"),
-};
-
-let dark: defaults = {
-  foreground: Color.hex("#ccc"),
-  editorBackground: Color.hex("#1E1E1E"),
-  editorForeground: Color.hex("#bbb"),
-  editorIndentGuideBackground: Color.hex("#404040"),
-  editorIndentGuideActiveBackground: Color.hex("#707070"),
-  editorWhitespaceForeground: Color.hex("#e3e4e229"),
-};
-
-let hcDark: defaults = {
-  foreground: Color.hex("#fff"),
-  editorBackground: Color.hex("#000"),
-  editorForeground: Color.hex("#FFF"),
-  editorIndentGuideBackground: Color.hex("#FFF"),
-  editorIndentGuideActiveBackground: Color.hex("#FFF"),
-  editorWhitespaceForeground: Color.hex("#e3e4e229"),
-};
-
-let getDefaults = uiTheme =>
-  switch (uiTheme) {
-  | "vs" => light
-  | "vs-light" => light
-  | "vs-dark" => dark
-  | "hc-black" => hcDark
-  | _ => dark
-  };
-
-type editorGutter = {
-  background: Color.t,
-  modifiedBackground: Color.t,
-  addedBackground: Color.t,
-  deletedBackground: Color.t,
-  commentRangeForeground: Color.t,
-};
-
-type t = {
-  foreground: Color.t,
-  editorBackground: Color.t,
-  editorActiveLineNumberForeground: Color.t,
-  scrollbarSliderBackground: Color.t,
-  scrollbarSliderActiveBackground: Color.t,
-  listActiveSelectionBackground: Color.t,
-  listActiveSelectionForeground: Color.t,
-  listFocusBackground: Color.t,
-  listFocusForeground: Color.t,
-  listHoverBackground: Color.t,
-  listHoverForeground: Color.t,
-  menuBackground: Color.t,
-  menuForeground: Color.t,
```

Comment:
Wow, exciting to see this all go away and be replaced with your new strategy. Nice work @glennsl !